### ISSUE TITLE
add support for JSON config

### ISSUE THREAD

**Issue by vvbandeira** [The-OpenROAD-Project/OpenLane-MPW-CI#128]
OpenLane now uses JSON to set configs instead of tcl files.

Enable back:
```
 //'mpw7_microwatt mpw7_Microwatt_FP_DFFRFile',//json format
 //'mpw7_microwatt mpw7_Microwatt_multiply_add',//json format
```

**Comment by stefanottili** [comment]
@vvbandeira, is there any plan on getting the OpenLane-MPW-CI working again or is this considered dead ?

**Comment by maliberty** [comment]
Effectively dead.  What's your interest?

**Comment by stefanottili** [comment]
I just saw that it was alive til recently, what killed it ?On Oct 18, 2024, at 19:21, Matt Liberty ***@***.***> wrote:﻿
Effectively dead.  What's your interest?

**Comment by maliberty** [comment]
I don't think it has run for a long time (@vvbandeira am I mistaken?).   In any case, why the interest?

**Comment by stefanottili** [comment]
I was looking for the (x86 ?) macOS compile logs that I remember to have seen, saw that the macOS check-in built was disabled (why ?), found the GitHub CI project linked from the OR projects, saw that it passed til a year and a half ago and was wondering why it was abandoned…

**Comment by maliberty** [comment]
PII is no longer involved in the MPW program (nor does it exist outside of chip ignite).   Mac was never supported and our only CI machine failed.
